addons/website/migrations/11.0.1.0/post-migration.py

```diff
@@ -68,6 +68,28 @@ def add_website_homepages(env):
     )
 
 
+def update_menu_urls(env):
+    """Update some menu URLs that must change."""
+    # Fix menus that include the "website." prefix
+    menus = env["website.menu"].search([
+        ("url", "=like", "/page/website.%"),
+    ])
+    for menu in menus:
+        menu.url = menu.url.replace("website.", "", 1)
+    # Fix menus pointing to "/page/homepage"
+    menus = env["website.menu"].search([
+        "|", "|", "|", "|", "|",
+        ("url", "=", "/page/homepage"),
+        ("url", "=", "/page/homepage/"),
+        ("url", "=like", "/page/homepage?%"),
+        ("url", "=like", "/page/homepage/?%"),
```

Comment:
For this case you will have as result `//?%`

Reply:
Odoo always autoremoves the trailing `/` (no matter if the controller exists or not) so there is zero chances that a user copies and pastes this link there, and when using autofill Odoo doesn't provide answers that end with `/`.

Taking that into account, I just removed the searches that finished in `/`. There's really so low chance those exist, and even more in a page such as homepage, which is generated by XML data.